Dockerfile

```diff
@@ -7,7 +7,9 @@ ENV BUILD_HOME=/go/src/github.com/ubclaunchpad/inertia/daemon/web
 ADD ./daemon/web ${BUILD_HOME}
 WORKDIR ${BUILD_HOME}
 # Build and minify client.
-RUN npm install --production
+RUN if [ ! -d "node_modules" ]; then \
```

Comment:
What if node_modules is out of date?

Reply:
Same problem with `dep` as well, where I skip the dependency install if `vendor` is already there. There's a bit of a risk, but I think the build time improvements is worth it for dev (`npm install` + `dep ensure` together can take more than a few minutes from home 😅 )

Plus, releases are built in a sterile environment on Travis with fresh `node_modules`, so I don't *think* it'll be a problem 🤔

Reply:
You can try adding a step that only copies over the package.json only, then running npm install, and then copying over your source files. If I'm understanding Docker correctly ( 😅), your node_modules + package.json will be cached in a layer and npm install will only run if package.json has changed rather than anything in your source code.

Reply:
Nice! A bit better than the current approach in that docker will automatically keep the modules up to date it needed. I imagine something similar with dep can be done as well